src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+document.addEventListener('click', event => {
+  if (event.target.children[0]
+    && event.target.children[0].className !== 'tree') {
+    event.target.children[0].classList.toggle('hidden');
+  }
+});
```

Comment:
each hovered item should become bold (you can wrap each header text in a span tag, it has CSS styles already)